docs/how-to-guides/convert-changelog-to-table.md

```diff
@@ -10,113 +10,139 @@ keywords: changelog, table, materialize
+You have a topic or a stream of events that represent a series of changes, known as a changelog. You want a view of the data that reflects only the last change for each key. Because ksqlDB represents change over time using tables, you need a way to convert your changelog into a table. This is broadly called *materializing* a changelog into a table.
 
-## In action
+## Materializing a changelog topic
 
-```sql
-CREATE TABLE t1 AS
-    SELECT k,
-           LATEST_BY_OFFSET(v1) AS v1,
-           LATEST_BY_OFFSET(v2) AS v2,
-           LATEST_BY_OFFSET(v3) AS v3
-    FROM s1
-    GROUP BY k
-    EMIT CHANGES;
-```
-
-## Materializing a changelog stream
-
-In ksqlDB, you derive new tables by aggregating other streams and tables. To create a table that reflects the latest values for each key, use the `LATEST_BY_OFFSET` aggregation.
-
-Begin by telling ksqlDB to start all queries from the earliest point in each topic.
+If you have a `changelog` topic, and you want a view of the data that reflects the latest values for each key then simply create a table with the `changelog` topic using the `CREATE SOURCE TABLE` statement.
 
+Let's say that you have the following data in your `changelog` topic where the first row is the record that has the earliest offset and the last row is the record that has the latest offset:
 ```sql
-SET 'auto.offset.reset' = 'earliest';
++------+------+------+------+
+|K     |V1    |V2    |V3    |
++------+------+------+------+
+|k1    |0     |a     |true  |
+|k2    |1     |b     |false |
+|k1    |2     |c     |false |
+|k3    |3     |d     |true  |
+|k2    |4     |e     |true  |
 ```
+Note that there are 2 records for the keys `k1` and `k2` in the `changelog` topic.
 
-Make a stream `s1` with four columns. `k` represents the key of the table. Rows with the same key represent information about the same entity. `v1`, `v2`, and `v3` are various value columns.
-
+Make a table `latest_view` with four columns. `k` represents the key of the table. Rows with the same key represent information about the same entity. `v1`, `v2`, and `v3` are various value columns.
 ```sql
-CREATE STREAM s1 (
-    k VARCHAR KEY,
+CREATE SOURCE TABLE latest_view (
+    k VARCHAR PRIMARY KEY,
     v1 INT,
     v2 VARCHAR,
     v3 BOOLEAN
 ) WITH (
-    kafka_topic = 's1',
+    kafka_topic = 'changelog',
     partitions = 1,
-    value_format = 'avro'
+    value_format = 'JSON'
 );
 ```
 
-Insert some rows into `s1`, repeating entries for some of the keys.
+Now, you can view the latest values for each key in your `changelog` topic by issuing a pull query against the `latest_view` table that we just created above:
```

Comment:
Might be good to mention, that if there is a lot of data in the topic, it might take a while until the table contains the latest data -- pull query are accepted right way but might return "old" data as long as the end of the input topic is not reached.